src/python/cli_new/lib/cli/mesos.py

```diff
@@ -114,15 +113,16 @@ def get_container_id(task):
         " Please try again.")
 
 
-def get_tasks(master, query=None):
+# pylint: disable=dangerous-default-value
```

Comment:
Maybe instead of disabling the pylint warning which is legitimate it'd be better to default `query=None` and in the function's body do:
```
if query is None:
    query = {'order':'asc'}
```

Reply:
done